cmd/kueuectl/app/list/list_workload_test.go

```diff
@@ -0,0 +1,348 @@
+wl1               j1         lq1          cq1            PENDING                       60m
+`,
+		},
+		"should print workload list with label selector filter (short flag)": {
+			args: []string{"-l", "key=value1"},
+			objs: []runtime.Object{
+				utiltesting.MakeWorkload("wl1", cmdtesting.DefaultNamespace).
+					OwnerReference(schema.GroupVersionKind{Group: "batch", Version: "v1", Kind: "Job"}, "j1", "test-uid").
+					Queue("lq1").
+					Active(true).
+					Admission(utiltesting.MakeAdmission("cq1").Obj()).
+					Creation(testStartTime.Add(-1*time.Hour).Truncate(time.Second)).
+					Label("key", "value1").
+					Obj(),
+				utiltesting.MakeWorkload("wl2", cmdtesting.DefaultNamespace).
+					OwnerReference(schema.GroupVersionKind{Group: "ray.io", Version: "v1", Kind: "RayJob"}, "j2", "test-uid").
+					Queue("lq2").
+					Active(true).
+					Admission(utiltesting.MakeAdmission("cq2").Obj()).
+					Creation(testStartTime.Add(-2*time.Hour).Truncate(time.Second)).
+					Label("key", "value2").
+					Obj(),
+			},
+			wantOut: `NAME   CRD TYPE   CRD NAME   LOCALQUEUE   CLUSTERQUEUE   STATUS    POSITION IN QUEUE   AGE
+wl1               j1         lq1          cq1            PENDING                       60m
+`,
+		},
+		"should print workload list with pending status": {
+			args: []string{},
+			objs: []runtime.Object{
+				utiltesting.MakeWorkload("wl1", cmdtesting.DefaultNamespace).
+					OwnerReference(schema.GroupVersionKind{Group: "batch", Version: "v1", Kind: "Job"}, "j1", "test-uid").
+					Queue("lq1").
+					Active(true).
+					Admission(utiltesting.MakeAdmission("cq1").Obj()).
+					Condition(metav1.Condition{
+						Type:   kueue.WorkloadQuotaReserved,
+						Status: metav1.ConditionFalse,
+						Reason: "Pending",
+					}).
+					Creation(testStartTime.Add(-1 * time.Hour).Truncate(time.Second)).
+					Obj(),
+				utiltesting.MakeWorkload("wl2", cmdtesting.DefaultNamespace).
+					OwnerReference(schema.GroupVersionKind{Group: "ray.io", Version: "v1", Kind: "RayJob"}, "j2", "test-uid").
+					Queue("lq2").
+					Active(true).
+					Admission(utiltesting.MakeAdmission("cq2").Obj()).
+					Condition(metav1.Condition{
+						Type:   kueue.WorkloadQuotaReserved,
+						Status: metav1.ConditionFalse,
+						Reason: "Pending",
+					}).
+					Creation(testStartTime.Add(-2 * time.Hour).Truncate(time.Second)).
+					Obj(),
+			},
+			wantOut: `NAME   CRD TYPE   CRD NAME   LOCALQUEUE   CLUSTERQUEUE   STATUS    POSITION IN QUEUE   AGE
+wl1               j1         lq1          cq1            PENDING                       60m
+wl2               j2         lq2          cq2            PENDING                       120m
+`,
+		},
+		"should print not found error": {
+			wantOutErr: fmt.Sprintf("No resources found in %s namespace.\n", cmdtesting.DefaultNamespace),
+		},
+		"should print not found error with all-namespaces filter": {
+			args:       []string{"-A"},
+			wantOutErr: "No resources found\n",
+		},
```

Comment:
Why?

Reply:
To tell user that we don't have resources, similarly as in kubectl.

Reply:
Do we have a test wit `-A` when we have workloads?